src/lib/aws/s3_manager.py

```diff
@@ -52,7 +52,7 @@ def read_file(self, s3_path: str) -> str:
             settings_data = response["Body"].read().decode("utf-8")
             return settings_data
         except ClientError as e:
-            if e.response["Error"]["Code"] == "NoSuchKey":
+            if e.response["Error"]["Code"] in ["NoSuchKey", "AccessDenied"]:
```

Comment:
Not sure if we need to handle File doesn't exist and Access Denied with the same message

Reply:
Good notice to make different messages, will do!

Reply:
Just want to leave also a little background for this change :) 
We are getting AccessDenied error when running code in Lambda, and NoSuchKey when debugging code locally, that's why both these keys should throw the same FileNotFound exception.

Reply:
Done